### PR TITLE
[systemtest] Change timeouts in resource classes and delete unused constants

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change

- Refactoring

### Description

In this PR I'm gonna change timeouts in resource classes, as I forgot to change it in previous PR. Other than that I removed unused constants, as we want to use `ResourceOperation` methods to determine the right timeout for each resource.

Also I added fix for `logCurrentResourceStatus` -> the NPE appeared at some tests if the CR wasn't created and the `T customResource` was empty (`null`). The problem is that we don't see that test failed on resource creation (timeout), but on this NPE. I think this will make it clearer.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only

File: systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceOperation.java [hunk 1]
```diff
@@ -42,7 +46,7 @@ public static long getTimeoutForResourceReadiness(String kind) {
                 timeout = Duration.ofMinutes(4).toMillis();
                 break;
             default:
-                timeout = Duration.ofMinutes(2).toMillis();
+                timeout = Duration.ofMinutes(3).toMillis();
```
Comment (see-quick) [inline systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceOperation.java:49]: Why this change ? :) 2 minutes were not enough?

File: systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaMirrorMakerResource.java [hunk 2]
```diff
@@ -24,6 +25,7 @@
 
 public class KafkaMirrorMakerResource {
     public static final String PATH_TO_KAFKA_MIRROR_MAKER_CONFIG = "../examples/mirror-maker/kafka-mirror-maker.yaml";
+    private static final long READINESS_TIMEOUT = ResourceOperation.getTimeoutForResourceReadiness();
```
Comment (see-quick) [inline systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaMirrorMakerResource.java:28]: Are you using this instance variable only inside `deployKafkaMirrorMaker` method?

File: systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceOperation.java (hunk 1 above)
Comment (im-konge) [reply systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceOperation.java:49]: I saw that in some cases it wasn't enough, by default it was set to 3 minutes, so I just changed it back.

File: systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaMirrorMakerResource.java (hunk 2 above)
Comment (im-konge) [reply systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaMirrorMakerResource.java:28]: Yes, but it's done like this if we want to add something else, otherwise I can add it straight to the `waitFor` -> and also to match the pattern to have it like this in all resource classes
